### PR TITLE
use Logging 5.9 in OCP 4.16 jobs

### PR DESCRIPTION
**Pull request by anpingli**

### CODE REVIEW

File: ci-operator/config/openshift-logging/extended-logging-tests-interop/openshift-logging-extended-logging-tests-interop-main__cluster-logging-ocp4.16-lp-rosa-classic.yaml [hunk 1]
```diff
@@ -18,11 +18,11 @@ base_images:
   tests-private:
     name: tests-private
     namespace: ci
-    tag: "4.16"
+    tag: "4.15"
   tests-private-ui:
     name: tests-private-ui
     namespace: ci
-    tag: "4.16"
+    tag: "4.15"
```
Comment (anpingli) [inline ci-operator/config/openshift-logging/extended-logging-tests-interop/openshift-logging-extended-logging-tests-interop-main__cluster-logging-ocp4.16-lp-rosa-classic.yaml:25]: I guess we need to run test-private 4.15 before Logging 6.0 is released in OCP 4.16

Comment (Sau1506mya) [reply ci-operator/config/openshift-logging/extended-logging-tests-interop/openshift-logging-extended-logging-tests-interop-main__cluster-logging-ocp4.16-lp-rosa-classic.yaml:25]: Okay, we will first test in this PR.

**Comment by anpingli** [comment]
/pj-rehearse periodic-ci-openshift-logging-extended-logging-tests-interop-main-cluster-logging-ocp4.16-lp-rosa-classic-cluster-logging-operator-rosa-aws

**Comment by Sau1506mya** [comment]
/pj-rehearse periodic-ci-openshift-logging-extended-logging-tests-interop-main-cluster-logging-ocp4.16-lp-rosa-classic-cluster-logging-operator-rosa-aws

**Pull request merged**